sample/src/androidTest/java/com/kaspersky/kaspressample/configurator_tests/enricher_tests/enrichers/LoggingMainSectionEnricher.kt

```diff
@@ -0,0 +1,19 @@
+package com.kaspersky.kaspressample.configurator_tests.enricher_tests.enrichers
+
+import com.kaspersky.kaspressample.configurator_tests.enricher_tests.data.EnricherTestData
+import com.kaspersky.kaspresso.enricher.MainSectionEnricher
+import com.kaspersky.kaspresso.testcases.core.testcontext.TestContext
+import com.kaspersky.kaspresso.testcases.models.info.TestInfo
+
```

Comment:
I think it's this feature is an ideal candidate to be presented in the documentations =)

Reply:
Ok, I'll add documentation into `Kaspresso configurator.md`